plugins/hydra_joblib_launcher/example/conf/config.yaml

```diff
@@ -0,0 +1,4 @@
+defaults:
+  - hydra/launcher: joblib
+
+task: 1
```

Comment:
can you add newline at the end? ( I know I am not doing it consistently but I would like to).

Reply:
Done